### ISSUE TITLE
Aya Neo 2021 Pro Wireless Card not Detected

### ISSUE THREAD

**Issue by MetalNeo** [ChimeraOS/chimeraos#278]
Aya Neo 2021 Pro's wireless card is not recognized both during installation and afterwards. The system is only able to connect to the Internet via a USB tether/Ethernet adapter.

The wireless card is detected normally in stock Arch Linux, but not within GamerOS.

**Comment by alkazar** [comment]
Strange, I have the non-pro Aya Neo and networking works fine.

What is the output of `lspci -v | grep Network` ?

Could just be your particular device needs a newer kernel.

**Comment by MetalNeo** [comment]
`01:00.0 Network controller: MEDIATEK Corp. Device 7961`

I think that Aya actually put a different wireless chipset inside of some of the newer units or perhaps the Pro units specifically due to an issue getting a hold of the original network cards. It may be because of that.

**Comment by alkazar** [comment]
That is the same as I have in my non-Pro. So I don't understand why it
wouldn't work.

**Comment by alkazar** [comment]
If you get a chance it might help if you can grab the entire "Network" section of `lspci -v` on Arch (where it is working) and ChimeraOS (where it is not working).

**Comment by MetalNeo** [comment]
OK. A few developments:

1. I appear to get the same output for lspci -v in both Arch and in ChimeraOS. I'll transcribe it if necessary, but I'm assuming it's the same as yours.
2. I decided to test network connectivity in the installer again, and I was surprised to see that it appears to be working now. No errors like before. I was able to reinstall without tethering.
3. Upon booting up the freshly reinstalled ChimeraOS, the system appears to have working wireless, but it is now suffering from connectivity/performance issues. Steam will begin to download updates, but will download at a rate of bytes per second. For reference, my home network is a 5GHz network with no password set (don't worry, I have no neighbors to worry about leeching/breaching the network).
